automaxlair/MaxLairInstance.py

```diff
@@ -192,7 +191,7 @@ def outline_region(
     def outline_regions(
         self,
         image: Image,
-        rects: Container[Rectangle],
+        rects: List[Rectangle],
```

Comment:
How come you decided to use lists instead of tuples? I'm not sure one it more correct so I'm wondering what your reason is.

Reply:
Aren't tuples fixed size ? The reason why I used list is that mypy was complaining about container and I think tuples are fixed size (while we have a variable number of rectangles)

Reply:
Tuples can be any length but you can't change them after they are created. In this case we don't need to add or remove the number of rectangles so a tuple is fine.
Can we try using Iterable as the type hint (as opposed to Container or List)? That should accept either a tuple or a list since it doesn't actually matter.